Dev-Corner/slides/tool_integration.html

```diff
@@ -59,103 +59,1315 @@
+                              Like `command` allow cheetah code<br/>
+                              <br/>
+                              <ul>
+                                <li>Create a config file</li>
+                                <li>Create a whole script or a module</li>
+                              </ul>
+                            </div>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### `configfiles`
+
+                            ```xml
+                            <command><![CDATA[ mb $script_nexus  ]]></command>
+
+                            <configfiles>
+                              <configfile name="script_nexus"><![CDATA[
+                            set autoclose = yes;
+                            execute $input_data;
+                            #if str( $data_type.type ) == "nuc“
+                                      lset nst=$data_type.lset_params.lset_Nst;
+                            #end if
+                            mcmcp ngen=$mcmcp_ngen;
+                            mcmc;
+                            quit
+                              ]]></configfile>
+                            </configfiles>
+
+                            ```
+
+                            ```bash
+                            set autoclose = yes;
+                            execute dataset_42.dat;
+                            lset nst=2 ;
+                            mcmcp ngen=100000;
+                            mcmc;
+                            quit
+                            ```
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### `command` > Reserved Variables
+
+                            ```xml
+                            <command><![CDATA[
+                              ${input_foobar.ext}
+
+                              --search_pfam "${ filter( lambda x: str( x[0] ) == str( $search.pfam_db ), $__app__.tool_data_tables[ 'pfam_databases' ].get_fields() )[0][-1] }"
+
+                              library $__app__.config.user_library_import_dir/$__user_email__/$inputs.library
+
+                              #if isinstance( $input1.datatype, $__app__.datatypes_registry.get_datatype_by_extension('gff').__class__):
+                            ]]></command>
+                            ```
+                            <small>[Reserved Variables List](https://docs.galaxyproject.org/en/latest/dev/schema.html#tool|command)</small>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### `macro`
+
+                        <img src="../images/macro.png" alt="macro"/>
```

Comment:
The image doesn't appear for me
+ maybe a word about tool suites? this is where macros are very useful